impexp-client/src/main/java/org/citydb/gui/modules/kml/view/KmlExportPanel.java

```diff
@@ -377,10 +367,10 @@ private void updateComponentUI() {
 
     public void doTranslation() {
         browseButton.setText(Language.I18N.getString("common.button.browse"));
-        gmlIdPanel.setTitle(Language.I18N.getString("kmlExport.label.singleBuilding"));
+        gmlIdPanel.setTitle(Language.I18N.getString("filter.border.attributes"));
```

Comment:
The keys "kmlExport.label.singleBuilding" and "kmlExport.label.noTiling" can be deleted from the language files.

Reply:
Done here 4016fcb